code/explainer.py

```diff
@@ -71,47 +72,38 @@ def predict(self, data):
     # ************* pd *************** #
 
     def pd(self, X, idv, grid):
-        """
-        numpy implementation of pd calculation for 1 variable 
-        
-        takes:
-        X - np.ndarray (2d), data
-        idv - int, index of variable to calculate profile
-        
-        returns:
-        y - np.ndarray (1d), vector of pd profile values
-        """
+
+        X = pd.DataFrame(data=X, index=range(X.shape[0]), columns=range(X.shape[1]))
+        # print(X)
+        explainer = dalex.Explainer(self.model, X, verbose=False)
+
+        ld_rf = explainer.model_profile(type = 'accumulated', variables=[idv], grid_points=len(grid), verbose=False)
```

Comment:
Wrzuciłem tak po prostu brutalnie ALE z dalexa i wygląda jakby działało chyba, że coś źle rozumiem

Reply:
Co to znaczy "jakby działało"?